ledger/participant-state/kvutils/tools/src/main/scala/ledger/participant/state/kvutils/tools/integritycheck/IntegrityChecker.scala

```diff
@@ -357,7 +363,7 @@ object IntegrityChecker {
 
   def run[LogResult](
       args: Array[String],
-      commitStrategySupportFactory: ExecutionContext => CommitStrategySupport[LogResult],
+      commitStrategySupportFactory: (Metrics, ExecutionContext) => CommitStrategySupport[LogResult],
```

Comment:
For improving readability, please consider adding a single method interface or a type def for creating a `CommitStrategySupport` instance from a `Metrics` and `ExecutionContext` pair.